### DISCUSSION TITLE
Code privacy

### DISCUSSION THREAD

**Discussion by lsmolinskiautiva** [community/community#13914] [Copilot Conversations] [answered]
Hi, I was wondering if I could use GitHub co pilot for a private repo with private code, does the code get sent over somewhere to be processed and analysed ? 

Please let me know

**Comment by davecheney** [comment, marked as answer]
Please see https://docs.github.com/en/github/copilot/about-github-copilot-telemetry

**Reply by 8ctopus** [reply]
Instead of linking to another doc, it would have been awesome had you answered the question. I've followed the link and could not find a clear answer about what happens with private repositories.
